### PR TITLE
Views training

### PR DESCRIPTION
**Pull request by liatsade**
https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/views-training/views.md

### CODE REVIEW

**Comment by liatsade** [comment]
@RachelBaram your turn :-)

File: views.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+This form give us some main View definitions: 
+* Set the View name and description - be kind and describe it shortly and clearly, so others can understand what this View shows.) 
+* In the ‘show’ field define what kind of information the View will show. This selection determines the main table in the DB that the View will use to retrieve data and **it cannot be changed later**.
+
+![](images/views/first_view_form.png)
+
+In this View we want to display a list of blog titles, so we select to show `Content` (nodes). Since we chose Content, we also get the option to select the content type. Although we know we need to choose the `Blog` content type, let’s leave it `All` for now, so we can practice how to define that filter in the edit View page.
+
+There is also the possibility to create a page and a block in this form. Our Blog list will display on a page, but we will create it at the next step (just for you to see how we define this in the View edit page.), So leave the page checkboxes unchecked.
```
Comment (RachelBaram) [inline views.md]: I don't see where you can create a page and block on this form.

**Comment by RachelBaram** [comment]
@liatsade I have only edited up to Displays. However, something that bothers me is the ability for someone to come back to the training to remind themselves of something they forget. The current way that the training is documented, there is no separation between the training scenario and the dry explanation of the different parts of the module. It would be nice if information that would be valuable to retrieve easily, was highlighted (maybe bullet points?)  I think this is an important point to think about for future documentation as well. What do you think? We can discuss offline. ;)

File: views.md (hunk 1 above)
Comment (liatsade) [reply views.md]: I added a screenshot.

**Comment by RachelBaram** [comment]
@liatsade Edits completed.

**Comment by liatsade** [comment]
@amitaibu ready for your review.

**Comment by liatsade** [comment]
@amitaibu as we talked, I merged it without your review.

**Pull request merged**